nunit.sln

```diff
@@ -133,249 +131,434 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "nunit.v2.driver", "src\NUni
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "nunit.v2.driver.tests", "src\NUnitEngine\nunit.v2.driver.tests\nunit.v2.driver.tests.csproj", "{B6D3AEAE-ECDA-4047-A9BA-CC6DA3EBEDF8}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mono.Addins", "src\NUnitEngine\mono-addins\Mono.Addins.csproj", "{91DD5A2D-9FE3-4C3C-9253-876141874DAD}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
The Any CPU configuration has been added again. We need to figure out a way to prevent it. I don't think removing the C++ projects will fix it. I think the x86 agent will also. We could just accept it is there, but we should modify NUnit.proj to build the correct configuration and there is no way to ensure developers pick the right one beyond making one the default.